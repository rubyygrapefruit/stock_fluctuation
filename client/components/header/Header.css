.my-badges {
  padding: 13px;
  text-transform: none;
  font-size: 13px;
  font-weight: 500;
  letter-spacing: 0.25px;
  line-height: 19px;
}
/* Styles for when market is Open */
.open-market-up {
  background: var(--primary-bglight);
  color: var(--primary-color);
}
.open-market-up:hover {
  background: var(--primary-bg);
}
.open-market-down {
  background: #fff5f3;
  color: #f45531;
}
.open-market-down:hover {
  background: #f45531;
  color: #fff;
}
.close-market-up {
  background: #182b27;
  color: var(--primary-color);
}
.close-market-up:hover {
  background: var(--primary-bg);
  color: #1b1b1d;
}
.close-market-down {
  background: #2d201f;
  color: #f45531;
}
.close-market-down:hover {
  background: #f45531;
  color: #1b1b1d;
}
