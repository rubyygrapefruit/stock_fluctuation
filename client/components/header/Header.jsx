import React, { Component } from 'react';
import './Header.css';

export class Header extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <nav className="uk-navbar-container uk-navbar-transparent" uk-navbar>
          <div className="uk-navbar-left">
            <ul className="uk-navbar-nav">
              <li>
                <a href="">
                  <span className="uk-badge top-badges">Computer Hardware</span>
                </a>
              </li>
              <li>
                <a href="">
                  <span className="uk-badge top-badges">100 Most Popular</span>
                </a>
              </li>
              <li>
                <a href="">
                  <span className="uk-badge top-badges">Computer Software</span>
                </a>
              </li>
            </ul>
          </div>
        </nav>
      </div>
    );
  }
}

export default Header;
