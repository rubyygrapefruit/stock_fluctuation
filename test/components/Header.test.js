import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../client/components/header/Header.jsx';

describe('<Header />', () => {
  test('it should render a <div />', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });

  test('should contain 3 spans', () => {
    const wrapper = shallow(<Header />);
    const spans = wrapper.find('span');
    expect(spans.length).toBeGreaterThan(0);
  });
});
