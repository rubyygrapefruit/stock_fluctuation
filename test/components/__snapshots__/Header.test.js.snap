// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> it should render a <div /> 1`] = `
<div>
  <nav
    className="uk-navbar-container uk-navbar-transparent"
    uk-navbar={true}
  >
    <div
      className="uk-navbar-left"
    >
      <ul
        className="uk-navbar-nav"
      >
        <li>
          <a
            href=""
          >
            <span
              className="uk-badge top-badges"
            >
              Computer Hardware
            </span>
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <span
              className="uk-badge top-badges"
            >
              100 Most Popular
            </span>
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <span
              className="uk-badge top-badges"
            >
              Computer Software
            </span>
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
